protocol/x/stats/keeper/keeper_test.go

```diff
@@ -197,12 +197,13 @@ func TestExpireOldStats(t *testing.T) {
 	})
 	windowDuration := tApp.App.StatsKeeper.GetWindowDuration(ctx)
 	// 5 epochs are out of the window
-	ctx = tApp.AdvanceToBlock(100, testapp.AdvanceToBlockOptions{
+	tApp.AdvanceToBlock(3, testapp.AdvanceToBlockOptions{
```

Comment:
@roy-dydx I didn't fully under stand this test case, so I just edit'ed the test set up to retain the previous behavior (setting specified `BlockTime` at block 3, then advance to 100)

Reply:
Does the test fail without this change?

It's supposed to simulate advancing to block `2` at `time.Unix(int64(1), 0).UTC()` and then advancing to block 100 at this specified time. I feel like it should work without the additional advance to 3?